Resources/views/page_head_style.html.twig

```diff
@@ -5,6 +5,8 @@
     <link rel="stylesheet" type="text/css" href="{{ asset_url }}"/>
 {% endstylesheets %}
 
+<link rel="stylesheet" type="text/css" href="/bundles/ezplatformuiassets/vendors/flag-icon-css/css/flag-icon.min.css" />
```

Comment:
It doesn't work. After rewriting paths to *.svg, they still refers to wrong (different) place.

@andrerom Second request will also occur in prod env. It's because there will be second <link> tag. `cssrewrite` must be used only for one css file, so I must put it into separate `{% stylesheet %}` block. Assetic doesn't have option to use filter only for one file in entire block.